File: TestSample/php/src/Speakers/KoSpeaker.php
<?php
declare(strict_types=1);

namespace HelloWorld\Speakers;

use HelloWorld\Person;

final class KoSpeaker extends Person
{
    public function __construct(string $name)
    {
        parent::__construct($name, "ko");
    }

    public function speak(): string
    {
        return "안녕, 세상!";
    }
}

File: TestSample/cpp/src/speakers/KoSpeaker.cpp
#include "KoSpeaker.h"

namespace helloworld::speakers {

KoSpeaker::KoSpeaker(std::string name)
    : Person(std::move(name), "ko") {}

std::string KoSpeaker::speak() const {
    return "안녕, 세상!";
}

}
